_archive/3-requirements.md

```diff
@@ -12,13 +12,13 @@ In order to reason about possible transports, `libp2p` uses [multiaddr](https://
 /ip6/fe80::8823:6dff:fee7:f172/tcp/4001/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
 
 # IPFS over uTP over UDP over IPv4 (UDP-shimmed transport)
-/ip4/162.246.145.218/udp/4001/utp/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+/ip4/203.0.113.0/udp/4001/utp/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
```

Comment:
I don't think there is value in touching any of the archived files. They are here for historic purposes only.

Reply:
Good point.